Dockerfile

```diff
@@ -14,10 +14,35 @@ RUN apt-get update -y && \
 
 RUN apt-get update -y && \
   apt-get upgrade -y && \
-  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig php7.0 php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu && \
+  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig libaio1 php7.0 php7.0-dev php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN mkdir /opt/oracle
+
+ADD oracle_instant_client_for_ubuntu_64bit/instantclient-basic-linux.x64-12.1.0.2.0.zip \
```

Comment:
I was thinking more about adding the extracted zip content to the rootfs folder, these layers are increasing the image size, just to get removed on a later step.

Reply:
OK, I am trying to change this, but I get this error:

error: File rootfs/instantclient_12_1/libociei.so is 104.47 MB; this exceeds GitHub's file size limit of 100.00 MB

Reply:
I would say focus on the QA and I will continue on that with another solution.

Reply:
OK, git-lfs doesn't work on the public repos. Getting the files from a remote server can lead to network issues , what we wanted to avoid when not requesting the zip files. 

Do you see any better solution to just unzip the files?

Reply:
@tboerger Any news about this?